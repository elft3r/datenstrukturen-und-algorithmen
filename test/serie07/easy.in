10
6 19
544 948 635 5 456 393
10 3 14 5 3 5 
8 41
194 333 628 554 127 359 80 238
4 19 2 13 13 18 7 10 
6 18
998 15 864 500 78 414
6 7 4 17 15 6 
6 17
153 755 589 103 915 126
1 8 14 4 5 7 
5 25
425 547 489 878 686
4 9 18 2 10 
7 43
503 326 545 594 758 218 298
17 17 20 9 13 7 20 
10 55
500 955 423 54 525 797 132 578 484 544
18 15 9 16 11 18 13 4 17 7 
9 54
341 113 483 480 407 143 432 642 518
12 3 12 15 15 10 1 16 4 
5 21
939 165 717 720 949
7 17 15 5 18 
7 18
956 114 522 322 368 704 393
4 12 1 8 1 12 14 
