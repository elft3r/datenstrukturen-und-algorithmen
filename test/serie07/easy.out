1948 1 2 5
1735 1 3 4 6
2276 1 3 6
1823 1 2 5
2536 1 2 4 5
1897 3 4 5
3399 2 5 6 8 10
2668 2 3 4 7 8 9
1659 1 4
2550 1 3 5 6
